test/govobject/types/trigger.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/* jshint unused: false */
+/* jshint latedef: false */
+var should = require('chai').should();
+var expect = require('chai').expect;
+var _ = require('lodash');
+var sinon = require('sinon');
+
+var bitcore = require('../../..');
+var BufferReader = bitcore.encoding.BufferReader;
+var BufferWriter = bitcore.encoding.BufferWriter;
+
+var Trigger = bitcore.GovObject.Trigger;
+var errors = bitcore.errors;
+
+describe('Trigger', function() {
+  it('should create new trigger', function() {
+    var trigger = new Trigger();
+
+    trigger.network = 'testnet';
+    trigger.event_block_height = 80760;
+    trigger.payment_addresses = 'yik5HAgVAgjH1oZKjcDfvcf22bwBNbSYzB';
```

Comment:
We should probably also test a trigger with multiple values, as well as some bad triggers.